package.json

```diff
@@ -8,7 +8,7 @@
   "scripts": {
     "test": "npm run lint && npm run unit-test",
     "unit-test": "tap test/lib -R=spec --timeout=300 --node-path ts-node --test-file-pattern '/\\.[tj]s$/'",
-    "lint": "tslint lib/**/*.ts",
+    "lint": "tslint -p tsconfig.json",
```

Comment:
Just a quick question - will it still be linting `.ts` files only? (https://github.com/snyk/nodejs-lockfile-parser/blob/4e08bcbdf8836229e03841cb5ae1d9cd2cc9a739/tsconfig.json#L14)

I tried to lookup this in the docs, but haven't found it.

Reply:
by default it will lint also .js, need to add something like `-e '**/*.js'` to skip `.js` - but not needed here